syft/frameworks/torch/tensors/interpreters/large_precision_tensor.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import torch
+
+from syft.frameworks.torch.overload_torch import overloaded
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+
+
+class LargePrecisionTensor(AbstractTensor):
+    """LargePrecisionTensor allows handling of numbers bigger than LongTensor
+
+    Some systems using Syft require larger types than those supported natively. This tensor type supports arbitrarily
+    large values by packing them in smaller ones.
+    If the original tensor is a PyTorch tensor it is not processed
+    """
+
+    def __init__(
+        self, owner=None, id=None, tags=None, description=None, precision=16, virtual_prec=0
+    ):
+        """Initializes a LargePrecisionTensor.
+
+        Args:
+            owner: An optional BaseWorker object to specify the worker on which
+                the tensor is located.
+            id: An optional string or integer id of the LargePrecisionTensor.
+            precision: The precision this tensor will be transformed to internally. In bits.
```

Comment:
what exactly is the difference between this and virtual prec

Reply:
I would change the naming as follows:
`precision` -> `internal_type`
`virtual_prec` -> `precision`